packages/common/src/components/popover/Popover.vue

```diff
@@ -15,15 +17,19 @@ import {
   onMounted,
   onUnmounted,
 } from '@vue/composition-api';
+import { createPopper } from '@popperjs/core';
 
 export default defineComponent({
   name: 'v-popover',
   setup() {
     const isOpen = ref(false);
     const root = ref<HTMLElement | null>(null);
+    let _popover = ref(null);
+    const trigger = ref<HTMLElement | null>(null);
+    const content = ref<HTMLElement | null>(null);
 
-    const toggle = () => (isOpen.value = !isOpen.value);
-    const open = () => (isOpen.value = true);
+    const toggle = () => ((isOpen.value = !isOpen.value), _popover.update());
```

Comment:
This method updates the position once it is "shown".

Reply:
I'd rather be explicit that two things are happening here, than have the function on one line. Thoughts:

```suggestion
    const toggle = () => {
      isOpen.value = !isOpen.value;
      _popover.update();
    };
```